tests/Fixer/ClassNotation/OrderedTypesFixerTest.php

```diff
@@ -660,7 +660,7 @@ public function testFixPhp82(string $expected, ?string $input = null, ?array $co
     }
 
     /**
-     * @return iterable<string[]|(string|null|array<string, string>)[]>
+     * @return iterable<(null|array<string, string>|string)[]|string[]>
```

Comment:
Isn't actually:

```suggestion
     * @return iterable<array<null|string|array<string, string>>>
```

??

I know it's not a part of this PR, but looking at it, seems overcomplicated. Not a something you should change, I would want to discuss it and create separate PR if needed.

Reply:
The best phpdoc here would be using array shape, ie. `iterable<array{ string, null|string, null|array<string, string> }>`.